README.md

```diff
@@ -319,15 +312,19 @@ being used to sign packages, in order to prevent cross-protocol attacks.
 
 #### Validating resources
 
-For a resource to be valid, its hash needs to match all of the hashes provided
-in the manifest, although, like
+For a resource to be valid, its hash needs to appear in the list for each
```

Comment:
Could you add a comment explaining how the hashes are mapped to resources? Is it based on the order in the array?